README.md

```diff
@@ -155,23 +173,22 @@ Assuming you already have everything needed to install the apps, this is what yo
    )
    ```
 
-
-5. OBS! for this step each cluster need to be up and running already.
+1. OBS! for this step each cluster need to be up and running already.
```

Comment:
Nit pick warning!
Is OBS a word you use in english?

Reply:
Note, is probably more appropriate :)
I can change.